extensions/azure/cosmos/contract-negotiation-store-cosmos/build.gradle.kts

```diff
@@ -17,14 +17,15 @@ plugins {
 }
 
 val cosmosSdkVersion: String by project
+val jodahFailsafeVersion: String by project
 
 dependencies {
     api(project(":spi"))
```

Comment:
does this really need the entire SPI or could we get away with `:spi:contract-spi`?

Reply:
Ditto, fixed